source/includes/fact-natural-language-querying-maxTimems.txt

```diff
@@ -0,0 +1,5 @@
+When using natural language querying, you may need to modify the 
```

Comment:
[wording suggestion]

Maybe "When you specify a natural language query..."

Reply:
Updated.